src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_RADIUS = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int FIGURES_AMOUNT = 5;
+    private static final int MAX_NUMBER = 25;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    private int getRandomNumber() {
```

Comment:
Private ones (methods, classes) should be **after** public ones.
```suggestion
    private int getRandomLength() {
```